src/components/Dashboard/ManageUsersPage/AddUserPage.scss

```diff
@@ -0,0 +1,7 @@
+.AddUser {
+  margin-left: 10%;
+  margin-right: 10%;
```

Comment:
Probably not a good idea to use percentages for margins or layout in general.

Reply:
We should probably look through the codebase and find places where this is happening and clean it up, I made an issue #229 for it